### PR TITLE
Added Jenkinsfile

### PR DESCRIPTION
**Pull request by autumnfound**
Signed-off-by: Martin Lowe <martin.lowe@eclipse-foundation.org>

### CODE REVIEW

**Comment by chrisguindon** [comment]
I think this a great starting point but the build is failing on jenkins:

https://foundation.eclipse.org/ci/webdev/job/api.eclipse.org-git/job/PR-2/1/console

File: src/main/k8s/production.yml [hunk 1]
```diff
@@ -28,6 +28,10 @@ spec:
           requests:
             cpu: 200m
             memory: 512Mi
+      volumes:
```
Comment (mbarbero) [inline src/main/k8s/production.yml:36]: Please check https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod for how to use secret inside pod's container. You need 2 things: a volume declaration (that reference the secret itself) and a volume mount to indicate where the volume should be mounted.

File: src/main/k8s/staging.yml [hunk 2]
```diff
@@ -28,6 +28,10 @@ spec:
           requests:
             cpu: 100m
             memory: 512Mi
+      volumes:
```
Comment (mbarbero) [inline src/main/k8s/staging.yml:36]: same as above

**Review by mbarbero — CHANGES_REQUESTED** [review]

**Review by chrisguindon — APPROVED** [review]
Looks good to me but I want to make sure that @mbarbero is ok with this before we merge!

**Review by mbarbero — APPROVED** [review]

**Pull request merged**